kapitan/cli.py

```diff
@@ -270,6 +270,23 @@ def build_parser():
         help='set schema cache path, default is "./schemas"',
     )
 
+    compile_parser.add_argument(
+        "--multiline-string-style",
+        type=str,
+        choices=["literal", "folded", "double-quotes"],
+        metavar="STYLE",
+        action="store",
+        default=from_dot_kapitan("compile", "multiline-string-style", "double-quotes"),
+        help="set multiline string style to STYLE, default is 'double-quotes'",
+    )
+
+    compile_parser.add_argument(
+        "--dump-null-as-empty",
```

Comment:
also this to `--yaml-dump-null-as-emtpy`?

Reply:
same here